en/physics/physics-rigidbody.md

```diff
@@ -10,16 +10,16 @@ Click the **Add Component -> Physics Component -> RigidBody** button at the bott
 
 | Properties      | Function explanation |
 | :-------------- | :------------------- |
-| Type            | The type of the rigid body, including **DYNAMIC**, **STATIC** and **KINEMATIC**, See below for details.
-| Mass            | The mass of the rigid body.                     |
-| AllowSleep      | Whether to allow the rigid body to enter sleep state.   |
-| Linear Damping  | Used to reduce the linear rate of rigid body. The larger the value, the slower the rigid body moves. |
-| Angular Damping | Used to reduce the rotation rate of rigid body. The larger the value, the slower the rigid body rotates. |
-| Use Gravity     | If enabled, the rigid body is affected by gravity.     |
-| Linear Factor   | It can affect the linear velocity change of the rigid body in each axis. The larger the value, the faster the rigid body moves. |
-| Angular Factor  | It can affect the rotation speed change of the rigid body in each axis. The larger the value, the faster the rigid body rotates. |
+| **Type**            | The type of the rigid body, including **DYNAMIC**, **STATIC** and **KINEMATIC**, See below for details.
+| **Mass**            | The mass of the rigid body.                     |
+| **AllowSleep**      | Whether to allow the rigid body to enter sleep state.   |
+| **Linear Damping**  | Used to reduce the linear rate of rigid body. The larger the value, the slower the rigid body moves. |
+| **Angular Damping** | Used to reduce the rotation rate of rigid body. The larger the value, the slower the rigid body rotates. |
+| **Use Gravity**     | If enabled, the rigid body is affected by gravity.     |
+| **Linear Factor**   | It can affect the linear velocity change of the rigid body in each axis. The larger the value, the faster the rigid body moves. |
+| **Angular Factor**  | It can affect the rotation speed change of the rigid body in each axis. The larger the value, the faster the rigid body rotates. |
 
-For the API interface of RigidBody, please refer to [Class RigidBody](../../../api/en/classes/physics.rigidbody.html).
+For more information, please refer to the [RigidBody API](https://docs.cocos.com/creator3d/api/en/classes/physics.rigidbody.html).
```

Comment:
Use the original relative path, it will open the link: <https://docs.cocos.com/creator/3.0/api/en/classes/physics.rigidbody.html>

Reply:
oops, I am not sure how that happened. I didn't recall changing the path. Relative is always better